exp/api/v1beta1/webhook.go

```diff
@@ -0,0 +1,117 @@
+/*
+Copyright 2022 The Kubernetes Authors.
```

Comment:
Would `webhook_helper.go` or `webhook_util.go` be a more accurate filename?

Reply:
With the expanded scope, moved all of this into `util/webhook/validator.go` where we get util/helper naming vibes for free.